src/utils/compileManager.ts

```diff
@@ -123,11 +124,12 @@ export class CompileManager {
                     .then((doc) => {
                         if (vscode.window.activeTextEditor) {
```

Comment:
replace this part with:
```typescript
for (const editor of vscode.window.visibleTextEditors) {
    if (editor.document.uri.toString()===fileUri.toString()) {
        editor.selections = editor.selections.map((sel, index) => {
            return index===0?
            new vscode.Selection(line,column+1,line,column+1)
            : sel;
        });
        break;
    }
}
```

Reply:
Update to the original pr already with line number replaced with line - 1 and additional `revealRange`